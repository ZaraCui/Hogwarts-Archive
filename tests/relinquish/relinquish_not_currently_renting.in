ADD COLLECTION tests/data/books_small.csv
ADD STUDENT Hermione Granger
RELINQUISH 100000 1004
EXIT

