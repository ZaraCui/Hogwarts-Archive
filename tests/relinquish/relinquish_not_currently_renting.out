user: 12 spellbooks successfully added.

user: Success.

user: Unable to return spellbook.

user: Ending Archive process.
